containers/roles/development/tasks/main.yml

```diff
@@ -0,0 +1,106 @@
+---
+  # Get facts such as database `adapter` and `uid` for the next tasks.
+- include_tasks: facts.yml
+  tags:
+    - always
+
+- name: docker-build image
+  docker_image:
+    name: "{{image_name}}"
+    tag: "{{image_tag}}"
+    path: images/develop/foreman
+    buildargs:
+      hostUID: "{{user_id}}"
+  tags:
+    - build
+
+# reset the bundle configs to download gems as specified
+- name: reset bundle configurations in {{foreman_src}}/.bunlde/config
+  file:
+    state: absent
+    path: "{{foreman_src}}/.bundle/config"
+  tags:
+    - install
+
+- name: install npm and gem packages
```

Comment:
Why not include this step in the build process? What is the use case for separate build and install steps?